usl_models/tests/flood_ml/flood_model_test.py

```diff
@@ -115,25 +45,23 @@ def test_convlstm_call():
     batch_size = 4
     height, width = 100, 100
     storm_duration = 12
-    params = pytest_model_params()
+    params = model_params.test_model_params()
 
     # The FloodConvLSTM model expects the data to have been preprocessed, such
     # that it receives the full temporal inputs and a single flood map.
```

Comment:
I think this comment and the docstring is slightly outdated. Can you update?

Reply:
Done